templates/user_profile_page.hbs

```diff
@@ -220,119 +241,164 @@
     {{/is}}
 
     {{#isnt current_filter.identifier 'activities'}}
-      <div class="container">
-        <section class="profile-section">
-
-          {{#if contributions}}
-
-            <header class="profile-section-header">
-              <h2 class="profile-section-title">{{t current_filter.identifier}}</h2>
-              {{#if sorters}}
-                <span class="profile-section-description">{{sorter_description}}</span>
-                <span class="profile-section-sorter dropdown">
-                  <button class="dropdown-toggle" aria-haspopup="true">
-                    {{current_sorter.label}}
-                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" focusable="false" viewBox="0 0 12 12" class="dropdown-chevron-icon">
-                      <path fill="none" stroke="currentColor" stroke-linecap="round" d="M3 4.5l2.6 2.6c.2.2.5.2.7 0L9 4.5"/>
-                    </svg>
-                  </button>
-                  <span class="dropdown-menu" role="menu">
-                    {{#each sorters}}
-                      <a href="{{url}}" aria-selected="{{selected}}" role="menuitem">
+      {{#is current_filter.identifier 'badges'}}
```

Comment:
can this block be outside the `isnt` block? If it is `badges` it is not `activities` :)